inst/@sym/private/cell2nosyms.m

```diff
@@ -0,0 +1,29 @@
+%% Copyright (C) 2016 Lagu
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+function z = cell2nosyms(x)
+
+  for i=1:length(x)
+      if (isa(x{i}, 'sym') || isa(x{i}, 'symfun'))
```

Comment:
minor nitpick but isn't a `symfun` also a `sym` so the "or" will never happen?  Also this block is intended too far.